cli/exchange/node.go

```diff
@@ -246,6 +246,8 @@ func NodeAddPolicy(org string, credToUse string, node string, jsonFilePath strin
 	}
 
 	// add/replce node policy
+	msgPrinter.Printf("Updating Node policy ...")
```

Comment:
"Updating Node policy and re-evaluating all agreements based on this policy. Existing agreements might be cancelled and re-negotiated."